features/container-queries.yml.dist

```diff
@@ -12,3 +12,51 @@ status:
     firefox_android: "110"
     safari: "16"
     safari_ios: "16"
+compat_features:
+  # baseline: low
+  # baseline_low_date: 2022-03-14
+  # support:
+  #   chrome: "52"
+  #   chrome_android: "52"
+  #   edge: "79"
+  #   firefox: "69"
+  #   firefox_android: "79"
+  #   safari: "15.4"
+  #   safari_ios: "15.4"
+  - css.properties.contain
+
+  # baseline: low
+  # baseline_low_date: 2022-07-26
+  # support:
+  #   chrome: "52"
+  #   chrome_android: "52"
+  #   edge: "79"
+  #   firefox: "103"
+  #   firefox_android: "103"
+  #   safari: "15.4"
+  #   safari_ios: "15.4"
+  - css.properties.contain.style
+
+  # baseline: low
+  # baseline_low_date: 2022-09-12
+  # support:
+  #   chrome: "105"
+  #   chrome_android: "105"
+  #   edge: "105"
+  #   firefox: "101"
+  #   firefox_android: "101"
+  #   safari: "16"
+  #   safari_ios: "16"
+  - css.properties.contain.inline-size
```

Comment:
These keys have no business being there and these comments make it obvious. Nice! Sent https://github.com/web-platform-dx/web-features/pull/1290